docs/src/doc/create-reusable-components/03_developing_components.stories.mdx

```diff
@@ -34,7 +34,16 @@ The recommended naming scheme for individual tokens is `wikit-<ComponentName>-[p
 
 ## Releasing a new component
 
-TODO
+Once your changes are merged and you want to release a new version of the components library you need to take the following steps:
+ - First create and push a new tag in compliance with [semantic versioning standards](https://semver.org/), e.g. `v1.2.3`
```

Comment:
npm has a pretty handy tool for that, I wonder if we should just instruct developers to use that  or at least link to the docs (if that was the original intention) https://docs.npmjs.com/cli/version

Reply:
The github workflow runs the tests before it publishes to npm, so that step is literally about the git command to create a tag. I guess we can add a link to this tool as well :)